navit/main.c

```diff
@@ -116,6 +131,14 @@ static void main_setup_environment(int mode) {
                     homedir="./";
                 val=g_strdup_printf("%s%s", homedir, val+1);
                 break;
+#ifdef HAVE_API_WIN32_BASE && !HAVE_API_WIN32_CE
+            case ';':
```

Comment:
Can we use the special character `~` also for Win32 as the homedir.
Win32 would thus use `NAVIT_USER_DATADIR`=`~\\navit`